src/dotty/tools/dotc/repl/InteractiveReader.scala

```diff
@@ -13,14 +13,13 @@ trait InteractiveReader {
 /** The current Scala REPL know how to do this flexibly.
  */
 object InteractiveReader {
-  /** Create an interactive reader.  Uses JLine if the
+  /** Create an interactive reader. Uses Ammonite if the
    *  library is available, but otherwise uses a
    *  SimpleReader. */
```

Comment:
Do we even need the SimpleReader anymore since the AmmoniteReader is always present in the dotty sources anyway?